src/chat/tools/getDockerContainers.ts

```diff
@@ -0,0 +1,150 @@
+import {
```

Comment:
Might be an opportunity in this file (happy to write a ticket and we can save for a later PR) to handle the case of no local connections by including a button to execute the command that creates one or the other, [similar to CCloud login button]( https://github.com/confluentinc/vscode/pull/1717/files#diff-afbe33c1b983f2e92340cc312638021c3133aa3a9eeceb3d3ce1b965489ced73R164) in the summarizing tools

Reply:
Should already happen based on the handling in here: https://github.com/confluentinc/vscode/blob/67de4e65091889f3902aeb6ef69585af5ec0f21f/src/chat/tools/getConnections.ts#L77-L86

...and the button + hinting here:
https://github.com/confluentinc/vscode/blob/67de4e65091889f3902aeb6ef69585af5ec0f21f/src/chat/tools/getConnections.ts#L171-L179

But there's a good chance we'll need to update descriptions and hinting anyway 😅